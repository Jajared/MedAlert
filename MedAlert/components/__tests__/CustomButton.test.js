import React from "react";
import renderer from "react-test-renderer";

import CustomButton from "../CustomButton";

describe("<CustomButton />", () => {
  it("has 1 child", () => {
    const tree = renderer.create(<CustomButton />).toJSON();
    expect(tree.children.length).toBe(1);
  });
  it("renders correctly", () => {
    const tree = renderer.create(<CustomButton />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
