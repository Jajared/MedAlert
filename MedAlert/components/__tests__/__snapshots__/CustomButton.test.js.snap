// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CustomButton /> renders correctly 1`] = `
<View
  accessibilityState={
    {
      "busy": undefined,
      "checked": undefined,
      "disabled": undefined,
      "expanded": undefined,
      "selected": undefined,
    }
  }
  accessibilityValue={
    {
      "max": undefined,
      "min": undefined,
      "now": undefined,
      "text": undefined,
    }
  }
  accessible={true}
  collapsable={false}
  focusable={false}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    {
      "flex": 2,
      "justifyContent": "center",
      "opacity": 1,
    }
  }
>
  <ViewManagerAdapter_ExpoLinearGradient
    proxiedProperties={
      {
        "colors": [
          4293750432,
          4293692521,
        ],
        "endPoint": [
          1,
          0.5,
        ],
        "locations": undefined,
        "startPoint": [
          0,
          0.5,
        ],
      }
    }
    style={
      {
        "borderRadius": 10,
        "paddingHorizontal": 80,
        "paddingVertical": 15,
      }
    }
  >
    <Text
      style={
        {
          "color": "black",
          "fontSize": 16,
          "fontWeight": "bold",
        }
      }
    />
  </ViewManagerAdapter_ExpoLinearGradient>
</View>
`;
